### PR TITLE
Implement initial build provision metrics

### PR DESCRIPTION
**Pull request by mmorhun**
This PR adds two prometheus metrics:
 - Time spent from the component creation till initial build pipeline created.
 - Time spent from the component creation till the component repository is provisioned for Pipelines-as-Code.

Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### CODE REVIEW

File: config/default/manager_auth_proxy_patch.yaml [hunk 1]
```diff
@@ -9,8 +9,13 @@ spec:
   template:
     spec:
       containers:
+      - name: manager
+        args:
+        - "--health-probe-bind-address=:8081"
+        - "--metrics-bind-address=127.0.0.1:8080"
+        - "--leader-elect"
```
Comment (Michkov) [inline config/default/manager_auth_proxy_patch.yaml:16]: should not be ports defined and probes set?

Comment (Michkov) [reply config/default/manager_auth_proxy_patch.yaml:16]: ok it's in config/manager/manager.yaml

**Comment by mmorhun** [comment]
After discussion with the team, changed buckets for the histograms, so now they are: `5, 10, 15, 20, 30, 60, 120, 300`

**Pull request merged**